client/src/Component/SMSModal.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable jsx-a11y/label-has-for */
+import { useState } from "react";
+import Modal from "react-bootstrap/Modal";
+import Button from "react-bootstrap/Button";
+
+function SMSModal(props) {
+    const [show, setShow] = useState(false);
+    const [disabled, setDisabled] = useState(false);
+    const handleClose = () => setShow(false);
+    const handleShow = () => setShow(true);
+
+
+    function handleSubmit () {
+      console.log(props.checkedCheckboxes);
+      handleClose();
+      }
+
+
+    return (
+      <>
+        <Button variant="primary" onClick={handleShow} disabled={props.checkedCheckboxes.length === 0}>
+          Send SMS
+        </Button>
+
+        <Modal show={show} onHide={handleClose} animation={false}>
+          <Modal.Header closeButton>
+            <Modal.Title>Send SMS</Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <form>
+              <label className="form-text text-muted" htmlFor="enterMessageText">Enter your message below</label>
```

Comment:
- The value of `htmlFor` has to match the `id` of the related input, so in this case the value should be `text` (to match with `id=text` of the textarea below) instead of `enterMessageText`
- Our label can be `Enter your message`—words describing direction like `below` is not usually preferred because they don't apply to everyone. A screen reader user can't see where `below` is, for example